ajax/comments.php

```diff
@@ -81,7 +83,58 @@
                     }
                 }
             }
-            echo ($tmpname["comment"] ?? '');
+            echo($tmpname["comment"] ?? '');
+
+            if (isset($_POST['withlink']) && isset($tmpname['link'])) {
+                echo "<script type='text/javascript' >\n";
+                echo Html::jsGetElementbyID($_POST['withlink']) . ".attr('href', '" . $tmpname['link'] . "');";
+                echo "</script>\n";
+            }
+            break;
+
+        case Group::getType():
+            if ($_POST['value'] == 0) {
+                $tmpname = [
+                    'link'    => $CFG_GLPI['root_doc'] . "/front/group.php",
+                    'comment' => "",
+                ];
+            } else {
+                $group = new \Group();
+                if (is_array($_POST["value"])) {
+                    $comments = [];
+                    foreach ($_POST["value"] as $groups_id) {
```

Comment:
I have trouble understanding how multiples values would be sent.
Isn't this a tooltip that is shown when you hover a group ?